Formula/mame.rb

```diff
@@ -55,6 +55,11 @@ class Mame < Formula
   fails_with gcc: "5"
   fails_with gcc: "6"
 
+  patch do
```

Comment:
Why is this needed and when can it be removed?

Reply:
I added a comment explaining that this should be removed with the next release.